packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -144,11 +168,15 @@ const CFDsListing = observer(() => {
                         return MT5_ACCOUNT_STATUS.MIGRATED_WITH_POSITION;
                     } else if (current_acc_status === 'migrated_without_position') {
                         return MT5_ACCOUNT_STATUS.MIGRATED_WITHOUT_POSITION;
+                    } else if (current_acc_status === 'under_maintenance') {
```

Comment:
Let's not hard code the value here

Reply:
this is following the existing code structure though... 🤔

Reply:
This was not the correct approach then. We can make these small improvements while doing new implementation.

Reply:
this will require refactoring throughout `Accounts` package as well
the values are response from BE which needs to be mapped for FE use
cc: @amina-deriv

Reply:
and `cashier` too